### PR TITLE
Created Documentation for the PaperSize class

### PR DESCRIPTION
**Pull request by MagnumBarrage**
edited the index.rst file in the doc folder and created the PaperSize.rst file in the module folder. 

Tested it with sphinx to see results. (Worked)

### CODE REVIEW

File: docs/modules/PaperSize.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The PaperSize Class
+------------------------
```
Comment (MartinThoma) [inline docs/modules/PaperSize.rst:2]:
```suggestion
The PaperSize Class
-------------------
```

I vaguely remember that there were warnings if the lines didn't match ... even if there are not, it looks nicer :-)

**Comment by MartinThoma** [comment]
Thank you for the PR :hugs:

**Comment by MartinThoma** [comment]
https://pypdf2.readthedocs.io/en/latest/modules/PaperSize.html

**Comment by MartinThoma** [comment]
@MagnumBarrage It might be helpful to add an example to that page how the class is used. Do you want to do that?

**Comment by MartinThoma** [comment]
In case you want to be added as a contributor https://pypdf2.readthedocs.io/en/latest/meta/CONTRIBUTORS.html - just let me know what to write / link there :-)

**Comment by xilopaint** [comment]
> @MagnumBarrage It might be helpful to add an example to that page how the class is used. Do you want to do that?

@MartinThoma Is the example added to the documentation?

**Comment by MartinThoma** [comment]
Nobody created one so far

**Comment by MagnumBarrage** [comment]
I can add an example later, I've just been preoccupied lately.

**Comment by SandroPanchame** [comment]
> @MagnumBarrage It might be helpful to add an example to that page how the class is used. Do you want to do that?

does it need to be on the same page as the documentation (under 'API Reference'), or would it need its own page (under 'User Guide'?

**Comment by MartinThoma** [comment]
I would expand the docstring of `add_blank_page`: https://pypdf.readthedocs.io/en/latest/modules/PdfWriter.html#pypdf.PdfWriter.add_blank_page or expand the docstring of the PaperSize class: https://pypdf.readthedocs.io/en/latest/_modules/pypdf/papersizes.html#PaperSize

I would not add a new page under "user guide" as I have doubts that this is a big enough topic. Too many doc pages might lead to the users not finding the relevant ones.

**Pull request merged**